test-js/config.test.js

```diff
@@ -0,0 +1,32 @@
+const util = require("node:util");
+const exec = util.promisify(require("node:child_process").exec);
+const { testUnsafe } = require("./helpers");
+
+describe("config param", () => {
+  test("eask archives -c", async () => {
+    const res = await exec("eask archives -c");
+    expect(res).toBeTruthy();
+  });
+
+  // TODO perhaps teardown the following two?
+  // TODO check in docker
+  testUnsafe("eask install -c", async () => {
+    const res = await exec("eask install -cp spinner ivy beacon company fuzzy");
```

Comment:
nit: `company-fuzzy` is one package and not two packages `company` and `fuzzy`. :)

Reply:
Nice, changed it back